concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md

```diff
@@ -4,12 +4,12 @@ By default, the **ColorBox** applies value after an end user clicks the "Apply"
 ##### jQuery
 
     <!--JavaScript-->
-    $(function() {
-        $("#colorBoxContainer").dxColorBox({
-            value: "#FF0000",
-            applyValueMode: 'instantly'
+        $(function() {
+            $("#colorBoxContainer").dxColorBox({
+                value: "#FF0000",
+                applyValueMode: 'instantly'
+            });
         });
-    });
```

Comment:
This example is misaligned now. Let's revert these changes.

Reply:
The changes are still present.

Reply:
fixed